onnxruntime/core/providers/openvino/contexts.h

```diff
@@ -21,11 +21,9 @@ struct GlobalContext {
   size_t num_of_threads;
   std::string device_type;
   std::string precision_str;
```

Comment:
We should add a depracation notice before we remove device_id

Reply:
Adding it in a seperate PR.